### ISSUE TITLE
feat: add layout tooltips

### ISSUE THREAD

**Issue by Ushie** [ReVanced/revanced-manager#1500]
### Feature description

Introduce the user to the layout of the app using tooltips, this can be dynamic in a way where if it detects that this user has previously used ReVanced Manager v1 (Flutter) it can respond appropriately, with "patching apps has been moved here" instead of "apps can be patched here" for example

### Motivation

Introduce the user to the new layout without any issues

### Additional context

https://github.com/skydoves/Balloon can be used for this, however a design and a set of "balloons" need to be decided before this can be implemented

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by PalmDevs** [comment]
Ideally it should look similar to the official extended tooltips, just with a little tail so it looks like a message bubble.

**Comment by oSumAtrIX** [comment]
To the user it may be unclear what this screen is:

![image](https://github.com/ReVanced/revanced-manager/assets/13122796/37bd5106-0fe6-4b6e-9301-2d34937eac20)

Some description, label, an initial dialog or similar should be present that explains to the user what this screen is,  its purpose and so on. Same goes for the apps screen.
